rocAL/rocAL/source/meta_data/meta_node_resize.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright (c) 2019 - 2020 Advanced Micro Devices, Inc. All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+#include "meta_node_resize.h"
+void ResizeMetaNode::initialize()
+{
+    _src_height_val.resize(_batch_size);
+    _src_width_val.resize(_batch_size);
+    _dst_width_val.resize(_batch_size);
+    _dst_height_val.resize(_batch_size);
+}
+void ResizeMetaNode::update_parameters(pMetaDataBatch input_meta_data, pMetaDataBatch output_meta_data)
+{
+    initialize();
+    std::cerr<<"\n check metanode update parameters ";
+    if(_batch_size != input_meta_data->size())
+    {
+        _batch_size = input_meta_data->size();
```

Comment:
An error is not required here?

Reply:
I followed the same template which has been used in other meta nodes. This is how its been handled in all metanodes